tests/pytests/test_search_params.py

```diff
@@ -292,7 +294,7 @@ def test_tags(env):
 
 
 def test_numeric_range(env):
-    env = Env(moduleArgs = 'DEFAULT_DIALECT 2')
+    env = Env(moduleArgs = 'DEFAULT_DIALECT 5')
```

Comment:
Consider adding a for loop over dialects 2 and 5 (as you did before) for the part that is common, and then adding another section with the part relevant only for dialect 5. This may help with clarifying the difference between them later on.

Reply:
OK. Done. Thanks.